docs/index.md

```diff
@@ -1,22 +1,21 @@
-.. Veristand Custom Device Handbook documentation master file, created by
-   sphinx-quickstart on Tue May 11 05:21:49 2021.
-   You can adapt this file completely to your liking, but it should at least
-   contain the root `toctree` directive.
+
 
 # Welcome to Veristand Custom Device Handbook's documentation!
```

Comment:
Let's replace the content of this line with:

VeriStand Custom Device Handbook